lib/vagrant-service-manager/command.rb

```diff
@@ -266,21 +265,17 @@ def print_vagrant_box_version(script_readable = false)
       end
 
       def restart_service(service)
-
-        if SCCLI_SERVICES.include? service
-          # TODO : Handle the case where user wants to pass extra arguments to OpenShift service
-          command = "sudo sccli #{service}"
-        else
-          command = "sudo systemctl restart #{service}"
-        end
-
-        with_target_vms(nil, { single_target: true}) do |machine|
-          machine.communicate.execute(command) do |type, error|
-            if type == :stderr
-              @env.ui.error(error)
-              exit 126
-            end
+        errors = []
+        command = "systemctl restart #{service}"
```

Comment:
`vagrant service-manager restart k8s` should work and that needs to execute `sccli k8s` . This was working prior to this PR and it should work post this PR too.